src/main/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriod.java

```diff
@@ -61,12 +66,17 @@ public int getTotalMonths() {
      * subscription period or false otherwise.
      *
      * @param date a date to consider
-     * @return true if true if the date comes after the expiration date of this
+     * @return true if the date comes after the expiration date of this
      * subscription period or false otherwise.
      */
     public boolean hasExpired(Date date) {
-        // todo implement method
-        throw new NotImplementedException();
+        // This is implemented
+        //throw new NotImplementedException();
+        if(date.compareTo(endDate) > 0)
+            return true;
+        return false;
+
+
```

Comment:
always use { } with if blocks. To find out why this a good thing read the Java Coding Conventions document. We use those conventions in this class. http://www.oracle.com/technetwork/java/codeconventions-150003.pdf